docs/workforce-management-framework/topics/con-security-introduction.adoc

```diff
@@ -0,0 +1,49 @@
+[]
+= Introduction to {WFM-RC-NameShort} security features
+
+{WFM-RC-NameShort} server gives developers a choice one of the following authentication and authorisation providers
+link:http://www.keycloak.org/[Keycloak] and link:http://passportjs.org/[Passport.js].
+{WFM-RC-NameShort} is build around security interface, which abstracts from any security libraries.
+Both implementations can be swapped at any time without breaking internal api of {WFM-RC-NameShort} modules and reference applications.
+Users can also provide their own security implementation by implementing their own security interface.s
+
+Another security measures are incorporated into our reference applications.
+Please note that all link:https://expressjs.com/[express] based routes outside WFM modules will also need to use these security interfaces.
+
+== How to apply authorization using security interface
+You can choose Keycloak or Passport.js implementation but you can securing endpoints in the same way.
+You need to import link:++../../../api/{WFM-RC-Api-Version}/auth-passport/docs/interfaces/_src_auth_passportauth_.endpointsecurity.html++[EndpointSecurity]
+interface from passportAuth. Apply either implementation initialisation to the interface.
+You can then use this security interface protect method to guard routes with roles.
+
+== Passport.js
+Passport.js is a lightweight authentication middleware for Node.js that is extremely flexible and modular.
```

Comment:
Passport.js is a lightweight authentication middleware for Node.js that provides a comprehensive set of strategies to support authentication.

Better description of passport - "extremely flexible and modular" is stated on the site and is not descriptive.

Reply:
removed   "extremely flexible and modular"